src/redis.js

```diff
@@ -65,6 +65,40 @@ export function setUserLang(user_id, lang) {
     });
 }
 
+export function getSong(guild_id, callback) {
+  return client.lindex(`guild_${guild_id}`, 0, (err, song) => {
```

Comment:
Why are these using callbacks? The entire redis client is promisified. You can instead use `client.lindexAsync`. Same goes for the others. :)